lib/src/profile/profile_controller.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:bedrock_flutter/src/auth/user_model.dart';
+import 'package:dio/dio.dart';
+import 'package:flutter/foundation.dart';
+
+import '../services/bedrock_service.dart';
+
+class ProfileController extends ChangeNotifier {
+  final BedrockService apiService = BedrockService();
```

Comment:
Can we inject the service via constructor? This is generally a good practice and it'll help a lot when adding tests